### DISCUSSION TITLE
USB OTG Doesn't power HL_PANv2... solved.

### DISCUSSION THREAD

**Discussion by gtxaspec** [gtxaspec/wz_mini_hacks#189] [General]
Curiously, using a USB OTG adapter or micro usb hub, doesn't power the PanV2 at all.  

Inspecting the unit inside, the connector has 5 wires coming from the base to the board.  if the ID pin is shorted for OTG, could this be preventing the camera from booting?

**Comment by gtxaspec** [comment]
Looks like I was right!  Removing the ID pin does allow the USB OTG cable / adapter / hub to power the device.  The USB devices work too!

If you have a PANv2 and you want to use USB OTG, for HUB, flash drive, ethernet adapter or whatever, see the attached photos.

Pop open the camera, then facing the T31 processor, you need to remove the 2nd red cable from the connector, I removed the pin and used heat shrink tubing, or go the easy way and cut it and tape the ends off without removing the connector. 

It is not needed for normal operation anyway.

**this will void your warranty**!

**Comment by gtxaspec** [comment]
![img2](https://user-images.githubusercontent.com/12115272/166132632-1ee93d4e-3630-49d9-818d-a366c7991646.jpg)
![img1](https://user-images.githubusercontent.com/12115272/166132631-072b5627-958d-44da-abc8-09d03a478506.jpg)

**Comment by capncybo** [comment]
Thank you for sharing this information, this makes me really want to purchase one.
